adhocracy4/static/TermsOfUseCheckbox.jsx

```diff
@@ -25,9 +31,11 @@ export const TermsOfUseCheckbox = (props) => {
       <label
         className="a4-termsofuse__checkbox-label"
         htmlFor={props.id}
-      >
-        {translated.termsOfUse}
-      </label>
+        dangerouslySetInnerHTML={{ __html: termsOfUseLabel }}
```

Comment:
just read about this prop. Seems to be fine to use in this case

Reply:
As we set it, I think it is fine!